apps/sample-app/README.md

```diff
@@ -1,11 +1,36 @@
 # Sample App
 
-Sample app leveraging teams js v2 for multi host experience
+The sample app leverages teams js v2 SDK for creating a multi host app experience. The sample app displays the usage of the following teams js v2 capabilties:
 
-(I will be working on this section below in the coming few weeks)
+- Get context (app.getContext)
```

Comment:
I think it's a bit confusing to have a mix of actual APIs (e.g. isSupported()) and just describing (e.g. Opening Mail Item)

Reply:
That makes sense! Would it be better to replace those with openMailItem(), startCallParams() etc, or remove those from the description itself?